utils/auth/jwtHelpers.ts

```diff
@@ -32,14 +34,16 @@ const USER_NOT_AUTHORIZED = 'User has not verified their identity.';
  * @returns NextApiHandler function.
  */
 export function withAnyUser<T>(
```

Comment:
since we have `withAuth`, we can rename this too, to make the meaning of `Auth` clear
```suggestion
export function withAuthUser<T>(
```